tracer/src/Datadog.Trace/PlatformHelpers/HostMetadata.cs

```diff
@@ -0,0 +1,149 @@
+﻿// <copyright file="HostMetadata.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+
+namespace Datadog.Trace.PlatformHelpers
+{
+    internal class HostMetadata
+    {
+        static HostMetadata()
+        {
+            TryGetKernelInformation(out var kernel, out var release, out var version);
```

Comment:
I liked your use of named parameters below to prevent confusion/ordering problems on the same typed parameters below. Small nit, but would you be interested in using it here too for protection from future us?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2097/commits/ba46d397484b7c73e2461971c73fcb9b1adc71bb